tools/generate-wicketd-api.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+./target/debug/wicketd openapi > openapi/wicketd.json
```

Comment:
Is this a convenience script, or is it called from some of our build machinery?

Reply:
Convenience script. There are others like it for nexus and sled agent.